### PR TITLE
Add ALTER TABLE limitation fragment

### PR DESCRIPTION
**Pull request by jhlodin**
Create a fragment discussing the multi-schema rename limitation and add to applicable connectors

### CODE REVIEW

**Comment by jhlodin** [comment]
@wendigo @hashhar What connectors in Trino support ALTER TABLE but with this limitation? I haven't found any explicit examples in our previous PRs.

**Comment by hashhar** [comment]
You can look for occurences of `case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:` in the code and those that `return false` for it are the ones which have this limitation.

By that logic I see the below have this limitation:

- Accumulo
- Elasticsearch
- MemSQL
- Oracle
- PostgreSQL
- SQL Server

**Comment by jhlodin** [comment]
Added to the SQL support sections of the connectors you listed above (those without SQL support sections, will add in later PRs), awaiting any updates on additional connectors

File: docs/src/main/sphinx/connector/alter-table-limitation.fragment [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
+^^^^^^^^^^^
+
+The connector does not support renaming tables across multiple schemas.
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/alter-table-limitation.fragment:4]: Maybe an example would help here.

```sql
-- This is supported
ALTER TABLE catalog.schema_one.table_one RENAME TO catalog.schema_one.table_two;
-- This is not supported
ALTER TABLE catalog.schema_one.table_one RENAME TO catalog.schema_two.table_two;
```

**Review by hashhar — APPROVED** [review]

**Review by hashhar — APPROVED** [review]
Thanks. Looks good to go.

**Review by mosabua — APPROVED** [review]
Looks good to me. To some degree I think this could apply to all connectors and therefore belong to the actual docs of the command.. but this is good as it stands now.

**Pull request merged**